### PR TITLE
[DO-1668] Actualize registers updates

### PR DESCRIPTION
**Pull request by Iva-khar**
Add method for actualizing update_at & total_records data in Register model
Add relevant api for KOATUU in fixture

### CODE REVIEW

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
when and who should run this method?

**Comment by Iva-khar** [comment]
> when and who should run this method?

1. After new project deployement;
2. After someone had run convertors manually or some shanges in registers without downloader.
3. Just now for bring register table up to date
4. Any time for ensure that register table is up to date

File: data_ocean/fixtures/register.json [hunk 1]
```diff
@@ -85,8 +85,8 @@
       "source_register_id": "dc081fb0-f504-4696-916c-a5b24312ab6e",
       "source_url_address": "https://data.gov.ua/dataset/d945de87-539c-45b4-932a-7dda57daf8d9/resource/296adb7a-476a-40c8-9de6-211327cb3aa1/download/koatuu.json",
       "source_api_address": "https://data.gov.ua/api/3/action/package_show?id=dc081fb0-f504-4696-916c-a5b24312ab6e",
-      "api_list": "/api/koatuu/",
-      "api_detail": "/api/koatuu/{id}/"
+      "api_list": "/api/koatuu-fourth-level/",
+      "api_detail": "/api/koatuu-fourth-level/{id}/"
```
[the remark this answers is not in the document]
Comment (Iva-khar) [reply data_ocean/fixtures/register.json:89]: Why? We don't have this API at all:x-special/nautilus-clipboard
copy
file:///home/ivan/Pictures/Screenshot%20from%202021-05-27%2014-39-16.png

**Comment by Iva-khar** [comment]
> how should run this method?

Management command added

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
I just checkout on your branch and take this error
```
django.core.exceptions.ImproperlyConfigured: Field name `update_at` is not valid for model `Register`.
```
One more
```
TypeError: handle() got an unexpected keyword argument 'verbosity'
```
Koattu register is not clickable at front
image

**Comment by Iva-khar** [comment]
> I just checkout on your branch and take this error
> 
> ```
> django.core.exceptions.ImproperlyConfigured: Field name `update_at` is not valid for model `Register`.
> ```
> 
> One more
> 
> ```
> TypeError: handle() got an unexpected keyword argument 'verbosity'
> ```
> 
> Koattu register is not clickable at front
> image

1. First error - from another PR - it disappiared after rebasing
2. Fixed management command
3. It fixed in front-end PR#135 https://github.com/3v-workspace/DataOcean/pull/135

**Comment by roman-tiukh** [comment]
image

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
image
Fixed

**Review by roman-tiukh — APPROVED** [review]

**Review by EmmaSamoilova — APPROVED** [review]

**Pull request merged**